Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -98,6 +98,9 @@ open class ChatChannelVC: _ViewController,
             setChannelControllerToComposerIfNeeded(cid: self?.channelController.cid)
             self?.messageComposerVC.updateContent()
         }
+
+        // Initial messages data
+        messages = Array(channelController.messages)
```

Comment:
Just a note: We should not expect messages to be available in setUp, specially with the coming up changes on Background Mappings.

Reply:
For now, these are used to render the initial offline messages only. Without this, it won't work with the current setup. If we change to background mappings , then we can think about how to improve this 👍